Dockerfile

```diff
@@ -1,3 +1,5 @@
+# syntax=docker/dockerfile:1
+
 FROM ghcr.io/linuxserver/baseimage-alpine:3.16
```

Comment:
Still using the 3.16 base...

Reply:
Resolved in https://github.com/linuxserver/docker-overseerr/pull/25/commits/b877e1ccbc21263e48f94f1c5d01fd00d879ffdf